README.md

```diff
@@ -1 +1,53 @@
-# Module1-Capstone-Project
\ No newline at end of file
+# Module1-Capstone-Project
+This is a Capstone Project for Module 1 website showcasing the Global Sabre Summit. Someof the features are replicated from Behance. The project is to showcase the mastery of what I have learned In Microverse's module 1.
+## Built With
+* HTML
+* CSS
+* JavaScript
+* Git
+* Behance
+
+## Live Demo 
+
+[Live Demo](https://carshy.github.io/Module1-Capstone-Project/)
+
+## Presentations
+[By Musoko Collins](https://www.loom.com/share/5f825eca31794c6ba5bf3033514690f0)
+## Getting Started
+
+To get a local copy up and running, follow these simple example steps.
+
+## Prerequisites
+* HTML, CSS, and JS
+* Git and Github
+* Terminal/cmd
+* VSCode or any other preferable IDE
+* 
+
+## Setup
+From the terminal, follow the following procedure:
+* Clone the repostory into your local machine
+use the command git clone
+* Navigate into the repostory with the command below
+cd module1 capstone project
+
+## Authors
+
+👤 **Collins Musoko**
+* Github: [@Carshy Collins](https://github.com/)
+* Linkedln: [@Collins Musoko](https://www.linkedin.com/feed/)
+## 🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/Carshy/Hello-microverse/issues)
+## Show Support
+Give a ⭐️ if love this project 
+# Acknowledgements
```

Comment:
- [ ] As requested by the previous code reviewer, which is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md) for this project, " `The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.` ", please kindly acknowledge the author of this design, in this acknowledgement section.